bin/cron/delete_accounts

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/db'
+require 'cdo/only_one'
+
+OPEN_ENDED_LEVEL_TYPES = %w(
+  Applab
+  FreeResponse
+  Gamelab
+  Weblab
+).freeze
+
+# Deletes all projects and project data associated with a user.
+# @param [Integer] user_id for which to delete projects and project data.
+def delete_projects_and_project_data(user_id)
+  # TODO(asher): Delete channel-backed projects.
+
+  GalleryActivity.where(user_id: user_id).delete_all
+end
+
+# @param [Integer] user_id for which to delete free response answers.
+def delete_open_ended_answers(user_id)
+  UserLevels.where(user_id: user_id).find_each do |user_level|
+    if OPEN_ENDED_LEVEL_TYPES.include? user_level.level.type
+      user_level.level_source.destroy
```

Comment:
Unless I'm misunderstanding something, deleting the level source indiscriminately seems like it isn't even an option, given how significant the impact would be on other users.

How bad would it be to simply remove the pointer from deleted user to level source and potentially have orphaned level sources floating around?